### DISCUSSION TITLE
Where to see deleted gists?

### DISCUSSION THREAD

**Discussion by tedzchow** [community/community#41943] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I've just deleted one of my gists. Stack Overflow recommended that I should contact support kindly but can't find the previous URL of that gist to request support. Shall I contact the support team with just the name of that gist or is there any page where I can deleted gists like deleted repositories in Settings/Repositories?

**Comment by radar-blip** [comment]
I cant find any place in settings that have to do with deleted Gists.
You will probably have to contact [Support](https://support.github.com/request)
I would think the name of the gist would be enough for them to go on (along with your username).

I also read the Stack Overflow discussion here: https://stackoverflow.com/questions/27300913/is-it-possible-to-recover-a-deleted-gist-on-github
(probably same one you looked at).
Anyway, contacting support is probably your best option.
Good luck!
